docs/UserGuide.md

```diff
@@ -50,158 +70,247 @@ Some example commands you can try:
+**Description:** Wanting to view all your shows at once? Display a list of shows in Trackermon's show list!
+
+**Format:** `list`
 
-Format: `list`
+**Example:** `list`
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
-### Requesting help URL: `help`
+### Exiting the program: `exit`
+
+**Description:** Wanting to exit the application? This simple command is what you are looking for!
 
-Shows a URL that redirects the user to Trackermon's user guide.
+**Format:** `exit`
 
-Format: `help`
+**Example:** `exit`
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
-### Finding a show: `find`
+### Viewing help: `help`
 
-**General Find**: Finds shows in Trackermon by matching the user's input across the name, status and tag parameters.
+**Description:** Are you new or confused with the commands? View the command summary and user guide through a quick pop up window!
 
-Format: `find <KEYWORD>`
-* Find shows with the specified `<KEYWORD>`.
-* The keyword refers to the input entered by the user after `find`.
-* The keyword **can be a word or number** such as hero, S1,...
-* There must be at **least one keyword** and it **must not be empty**.
+**Format:** `help`
 
-Examples:
-* `find shingeki` displays all the shows in the list that contain the keyword `shingeki` whether it is a name, status or tag.[put image here after ui update]
-* `find shingeki no kyojin` displays all the shows in the list that contain the keyword `shingeki` **or** `no` **or** `kyojin` whether it is a name, status or tag.[put image here after ui update]
+**Example & Output:** `help`
 
-**Precise Find** Finds shows in Trackermon by matching the user's input across name, status and tag parameters with prefixes.
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
-Format: `find n/[NAME] s/[STATUS] t/[TAG]`
-* **Within a single prefix**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all keywords that are input by the user in the specific prefix.
-* **Across multiple prefixes**, the find command will execute an **AND** search across Trackermon's show list and return all shows that match all the keywords across all prefixes.
-* There must be at **least one prefix field** and it **must not be empty**.
+[return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
-Examples:
-* `find n/shingeki s/watching t/seinen` displays all the shows in the Trackermon's show list that contain **shingeki** in NAME, **watching** in STATUS, and **seinen** in TAG. [put image here after ui update]
-* `find n/Shutter Island s/watching` displays all the shows in the Trackermon's show list that contains **Shutter Island** in NAME, and **watching** in STATUS. [put image here after ui update]
+---
+
+### Adding a show: `add`
+
+**Description:** Planning to watch a show and want to store its details? Add a show to Trackermon's show list!
```

Comment:
Really nitpicking this but should it be "Add the show to Trackermon's show list!" or "Add it to Trackermon's show list!" I feel like it would better convey the context provided in the first sentence!

Reply:
Yes I agree